Google.PowerShell/Provider/GoogleCloudStorageProvider.cs

```diff
@@ -672,6 +686,7 @@ public IContentWriter GetContentWriter(string path)
             request.UploadAsync();
             IContentWriter contentWriter = new GcsContentWriter(inputStream);
             TelemetryReporter.ReportSuccess(nameof(GoogleCloudStorageProvider), nameof(GetContentWriter));
+            BucketModels.Clear();
```

Comment:
As per-usual, assumne the next person reading the code isn't that smart. (Like me.)

Why are we clearing the bucket models? Is it just because we don't want to update the internal model with the results of the operation? If so, that's fine... but let's leave a brief comment to explain as much.